src/lib.rs

```diff
@@ -139,7 +139,8 @@ pub use gdk_ffi::enums::{
     WindowWindowClass,
     Gravity,
     DragAction,
-    DragProtocol
+    DragProtocol,
```

Comment:
This is why trailing commas are recommended so adding or removing an element doesn't touch other lines.